### PR TITLE
Word Styling

### PR DESCRIPTION
**Pull request by chan227**
User can now upload image, change word style and upload link in post and comment box

### CODE REVIEW

**Comment by chan227** [comment]
ready

**Comment by chan227** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -8,7 +8,9 @@ def index
     @users = User.sort_by_name.page(params[:page]).per(Settings.users.page.default_page)
   end
 
-  def show; end
+  def show
+    @posts = @user.posts.order(created_at: :desc).page(params[:page]).per(Settings.posts.page.max)
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb]: remove () and write scope for order

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -18,3 +18,13 @@
     </section>
   </aside>
 </div>
+
+<div class="container">
+  <% if @user.posts.any? %>
+    <h3>Posts (<%= @user.posts.count %>)</h3>
```
Comment (saobv-2223) [inline app/views/users/show.html.erb]: using I18n for this text

Comment (chan227) [reply app/views/users/show.html.erb]: done

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (chan227) [reply app/controllers/users_controller.rb]: done

**Comment by chan227** [comment]
ready

**Pull request merged**